cEP-0024.md

```diff
@@ -0,0 +1,96 @@
+# Implement GitMate automations and plugins for coala
+
+| Metadata |                                                       |
+| -------- | ----------------------------------------------------- |
+| cEP      | 24                                                    |
+| Version  | 1.0                                                   |
+| Title    | Implement Gitmate automations and plugins for coala   |
+| Authors  | Vamshi Krishna Bommerla <mailto:bommvams72@gmail.com> |
+| Status   | Proposed                                              |
+| Type     | Process                                               |
+
+## Abstract
+
+This cEP describes the details of the process of creating new automations and
```

Comment:
this doesn't describe "the process of creating new automations" (btw. "automations" isn't a valid plural, it's simply "automation" I think). This document describes improvements to the coala workflow and the necessary automation to realise it, no?